README.md

```diff
@@ -1,4 +1,79 @@
 # MyOme Variant Annotation Technical Challenge
 
-See [CHALLENGE-README.md](CHALLENGE-README.md) for instructions.  Please update this README.md as
-described in the challenge instructions.
\ No newline at end of file
+## Description
+This package takes input from a file of genome variations and looks up the specified annotations in the ensembl.org database via their REST service, and writes the output to a Tab Separated Values (TSV) file in the directory the user was in when executing the file. 
+
+The output file name format is 'annotated_data_\<timestamp\>.tsv'
+
+## Installation instructions
+
+Change directory to wherever you installed the variant_annotator.py file is and make it executable if it is not already:
```

Comment:
See notes on console scripts/entry points here: https://setuptools.pypa.io/en/latest/userguide/entry_point.html for a more package oriented approach.